Updated QS How-To Content

thripti opened the pull request:
I've updated the content. 

Pending work: Formatting to be done by TA. Need to finalize the styles to be used (For example, formatting for path name, variables, etc.)
Sudha to verify the code samples and update them as necessary.

Aravind-21-Test committed:
Update query.service.adoc

Added the updated content. TA needs to verify the formatting and make any necessary edits. Sudha to verify the code samples.

Aravind-21-Test committed:
Update query.service.adoc

Updated the Overview and fixed Formatting

Aravind-21-Test committed:
Update query.service.adoc

Modified styles and formats.

Aravind-21-Test committed:
Update query.service.adoc

Updated: Resources topic, fixed typos, and improved the content

thripti commented:
Kindly review the new changes: Added API Catalog to the Reference section; made minor edits to improve the content.

sv0330 reviewed (approved):
I observed that you filtered some scenarios from How-To. Overall, this looks good.

Aravind-21-Test committed:
Update query.service.adoc

Editorial Changes:

Teradata Database > Analytics database

Updated the Reference section to remove the reference to docs portal as API Spec is available in the new catalog portal.
